libpdbg/target.c

```diff
@@ -526,7 +533,12 @@ int ocmb_putscom(struct pdbg_target *target, uint64_t addr, uint64_t val)
 {
 	struct ocmb *ocmb;
 
-	assert(pdbg_target_is_class(target, "ocmb"));
+	assert(pdbg_target_is_class(target, "ocmb") || is_child_of_ody_chip(target));
+
+	if(is_child_of_ody_chip(target))
+	{
+		target = get_class_target_addr(target, "ocmb", &addr);
+	}
 
```

Comment:
this code should go away after we do refactor so add TODO: link to our refactor commit